apps/web/frontend/components/reportForm/userLabel.tsx

```diff
@@ -1,15 +1,67 @@
+import Image from 'next/image'
+
 interface UserLabelProps {
   name: string
   duration: number
+  image?: string
+  members?: { id: string; image?: string; name?: string }[]
+  isAllUsers?: boolean
```

Comment:
is this prop needed?

Reply:
Yohannes is going to use this prop where he does not use all users. That's why I added it. But now, I have removed it so that he can add it later.